ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -519,8 +519,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
-- as: aws-ipi-private-f28-disasterrecovery
-  cron: 44 8 5 * *
+- as: aws-ipi-private-f6-disasterrecovery
+  cron: 44 8 1,7,13,19,25 * *
```

Comment:
/cc @jianlinliu

Reply:
@geliu2016 can I Know any background to set a high test frequency for the dr testing, especially for old versions?

Reply:
@jianlinliu Mgr. team want to do dr test cover 4.11-4.15 every day, and 4.11 dr test will be deprecated after 4.15 released. cc: @wsun1 @xingxingxia

Reply:
Every day? @wsun1 @xingxingxia need so high frequency?

Reply:
The reason of caring about the frequency, because now we are struggling with the budget exceed issue, we need a thoughtful balance of cloud cost and test coverage.

Reply:
Weekly for each version at least, not daily for all of the versions. https://issues.redhat.com/browse/OCPQE-17626 , the related Jira task. Thanks!

Reply:
@jianlinliu adjusted 1 DR test per day, could you please review? thx